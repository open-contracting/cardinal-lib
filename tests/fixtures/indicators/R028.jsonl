{"ocid":"F","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"D"}]}]},"tender":{"procurementMethodDetails":"R028"}}
{"ocid":"missing tenderers","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"}},{"status":"disqualified","value":{"amount":1,"currency":"USD"}}]},"tender":{"procurementMethodDetails":"R028"}}
{"ocid":"same tenderers","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"T"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"T"}]}]},"tender":{"procurementMethodDetails":"R028"}}
{"ocid":"shuffled tenderers","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"A"},{"id":"Z"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"Z"},{"id":"A"}]}]},"tender":{"procurementMethodDetails":"R028"}}
{"ocid":"different decimals","bids":{"details":[{"status":"valid","value":{"amount":1.0,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1.1,"currency":"USD"},"tenderers":[{"id":"D"}]}]},"tender":{"procurementMethodDetails":"R028"}}
{"ocid":"different currencies","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"CAD"},"tenderers":[{"id":"D"}]}]},"tender":{"procurementMethodDetails":"R028"}}
{"ocid":"cancelled tender","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"D"}]}]},"tender":{"procurementMethodDetails":"R028","status":"cancelled"}}
{"ocid":"no price comparison","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"D"}]}]},"tender":{"procurementMethodDetails":"NPC"}}
{"ocid":"exclude procurement method details","bids":{"details":[{"status":"valid","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"V"}]},{"status":"disqualified","value":{"amount":1,"currency":"USD"},"tenderers":[{"id":"D"}]}]},"tender":{"procurementMethodDetails":"EXC"}}
