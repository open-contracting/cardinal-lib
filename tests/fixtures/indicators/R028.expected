{"OCID":{"F":{"R028":1.0}}}
